app/models/validations/shared_validations.rb

```diff
@@ -102,6 +103,15 @@ def inactive_status(date, resource)
     { scope: status, date: date&.to_formatted_s(:govuk_date), deactivation_date: closest_reactivation&.deactivation_date&.to_formatted_s(:govuk_date) }
   end
 
+  def tenancy_startdate_with_scheme_locations(record)
+    return unless record.respond_to?(:scheme)
```

Comment:
What would be a situation where record doesn't respond to scheme?

Reply:
Sales log tests brought up some errors, line 107 was the fix.

Reply:
I'm slightly worried by the idea that this is being called at all for sales logs - do you know why it is?

Reply:
Seems like this function failing those tests is not repeatable and this line 107 is not actually needed, so I've removed it.